From 2d36c72577d7dbe5400a7c5abafbc1d2ca84a4f5 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Mon, 15 May 2017 23:02:56 +0800
Subject: [PATCH 1783/5242] MLK-14895-2 ARM: dts: add usb otg1 ID pinctrl for
 i.mx6ul evk

commit  6f38885c98ed58356777de73cfc27d6961b26034 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add USB OTG1 port ID pin for imx6ul-14x14-evk.dts

Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-14x14-evk.dtsi |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
index 6e23a86..f92b4c8 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
@@ -390,6 +390,8 @@
 };
 
 &usbotg1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_usb_otg1_id>;
 	dr_mode = "otg";
 	srp-disable;
 	hnp-disable;
@@ -653,6 +655,11 @@
 		>;
 	};
 
+	pinctrl_usb_otg1_id: usbotg1idgrp {
+		fsl,pins = <
+			MX6UL_PAD_GPIO1_IO00__ANATOP_OTG1_ID	0x17059
+		>;
+	};
 
 	pinctrl_usdhc1: usdhc1grp {
 		fsl,pins = <
-- 
1.7.9.5

